OracleTuxedo/core/dockerfiles/buildDockerImage.sh

```diff
@@ -144,17 +122,12 @@ fi
 
 echo "====================="
 
-# Fix up the locations of things
-cp ${VERSION}/tuxedo${VERSION}.rsp .
-cp ${VERSION}/oraInst.loc .
-cp ${VERSION}/Dockerfile .
-
-docker build $PROXY_SETTINGS -t oracle/tuxedo:${VERSION} .
+docker build $PROXY_SETTINGS -t oracle/tuxedo:${VERSION} ${VERSION}/
 if [ "$?" = "0" ]
     then
 	echo ""
 	echo "Tuxedo Docker image is ready to be used. To create a container, run:"
-	echo "docker run -ti -v \${LOCAL_DIR}:/u01/oracle/user_projects oracle/tuxedo:${VERSION} /bin/bash"
+	echo "docker run -d -v \${LOCAL_DIR}:/u01/oracle/user_projects oracle/tuxedo:${VERSION} /bin/bash"
```

Comment:
Please do not ask users to run `/bin/bash`. This is a container, not a VM. The `CMD` or `ENTRYPOINT` script should run and that should provide the required functionality.